glTF-Toolkit.UWP.Test/glTF-Toolkit.UWP.Test.csproj

```diff
@@ -131,13 +131,13 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.0.7</Version>
+      <Version>6.2.0-Preview1-26502-02</Version>

```

Comment:
Any reason why we're using a preview version here?

Reply:
I just pushed the update button in NuGet; there is no reason I needed this package updated.